opencti-platform/opencti-front/src/private/components/common/files/FileLine.tsx

```diff
@@ -497,7 +497,7 @@ const FileLineComponent: FunctionComponent<FileLineComponentProps> = ({
   );
 };
 
-const FileLine = createFragmentContainer(FileLineComponent, {
+const FileLine = createFragmentContainer(FileLineComponent, { // objectMarking -> rel_object-marking.internal_id
```

Comment:
I don't get what this comment means

Reply:
It was meant to tell me how to implement the marking relation.
I just forgot to remove it.

Thanks.